### PR TITLE
mdd-split-changelog

### PR DESCRIPTION
**Pull request by mddifilippo89**
Updated changelog per platform

### CODE REVIEW

File: docfx/en/components/toc.json [hunk 1]
```diff
@@ -63,34 +63,28 @@
...
-        "href": "blazor-webassembly-how-to-read-and-write-excel-files-to-reduce-server-load.md",
-        "status": ""
-      },
-      {
-        "exclude": ["Angular", "React", "WebComponents", "Blazor"],
-        "name": "How To Customize Marker Display in Blazor Charts",
-        "href": "customize-marker-display-in-blazor-charts.md",
-        "status": ""
-      }
-    ]
```
Comment (HUSSAR-mtrela) [inline docfx/en/components/toc.json:89]: undo removal of these 3 topics
image

File: docfx/en/components/toc.json [hunk 2]
```diff
@@ -63,34 +63,28 @@
...
     "name": "Changelog",
-    "href": "general-changelog-dv.md",
+    "href": "general-changelog-dv-react.md",
+    "status": "UPDATED"
+  },
+  {
+    "exclude": ["Angular", "React", "Blazor"],
+    "name": "Changelog",
+    "href": "general-changelog-dv-wc.md",
     "status": "UPDATED"
```
Comment (HUSSAR-mtrela) [inline docfx/en/components/toc.json:88]: move these 3 nodes above the node for "general-changelog-dv.md" (Angular)

image

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: docfx/en/components/toc.json (hunk 1 above)
Comment (mddifilippo89) [reply docfx/en/components/toc.json:89]: see 
https://github.com/IgniteUI/igniteui-xplat-docs/pull/383/commits/6d6fce67106c04140fb7dae241e45beadabc6d62

File: docfx/en/components/toc.json (hunk 2 above)
Comment (mddifilippo89) [reply docfx/en/components/toc.json:107]: see 
https://github.com/IgniteUI/igniteui-xplat-docs/pull/383/commits/6d6fce67106c04140fb7dae241e45beadabc6d62

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**